manifests/daemon.pp

```diff
@@ -81,6 +81,9 @@
   Boolean $manage_service         = true,
   Hash[String, Scalar] $env_vars  = {},
   Optional[String] $env_file_path = $prometheus::env_file_path,
+  Boolean $export_scrape_job      = false,
+  String $scrape_host             = $facts['fqdn'],
```

Comment:
We can probably use this datatype? https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/fqdn.pp